AayushRawat.java

```diff
@@ -0,0 +1,31 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+public class Solution {
+
+	public static void main(String[] args) {
+		Scanner sc = new Scanner(System.in);
+		String text = sc.nextLine();
+		int q = sc.nextInt();
+		ArrayList<Integer> res = new ArrayList<>();
+		sc.nextLine();
+		while(q-- > 0) {
```

Comment:
Please use a separate method to get the output instead of writing the logic in the main method & run formatter to give proper indentation in your code.